### PR TITLE
Doc/add type flag to container registry documentation

### PR DESCRIPTION
**Pull request by AlexejPenner**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
When following the example I got an error message regarding the missing --type flag. Not entirely sure the default value is correct for the gcp docs

### CODE REVIEW

File: examples/kubeflow/setup.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 set -Eeo pipefail
 
 setup_stack () {
-  zenml container-registry register local_registry --uri=localhost:5000 || \
+  zenml container-registry register local_registry --uri=localhost:5000 --tpye=default|| \
```
Comment (htahir1) [inline examples/kubeflow/setup.sh:6]: Typo I think `--type`

File: examples/kubeflow/README.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ following four parts:
 
 ```bash
 # Make sure to create the local registry on port 5000 for it to work 
-zenml container-registry register local_registry --uri=localhost:5000
+zenml container-registry register local_registry --uri=localhost:5000 --type=default
```
Comment (htahir1) [inline examples/kubeflow/README.md:56]: Small nitpick: I would put it in the front of the command

**Review by htahir1 — CHANGES_REQUESTED** [review]
Just two small nit-picks but otherwise nice catch!

File: examples/kubeflow/setup.sh (hunk 1 above)
Comment (AlexejPenner) [inline examples/kubeflow/setup.sh:6]:
```suggestion
  zenml container-registry register local_registry  --type=default --uri=localhost:5000|| \
```

File: examples/kubeflow/README.md (hunk 2 above)
Comment (AlexejPenner) [inline examples/kubeflow/README.md:56]:
```suggestion
zenml container-registry register local_registry --type=default --uri=localhost:5000 
```

File: docs/book/guides/functional-api/deploy-to-production.md [hunk 3]
```diff
@@ -159,7 +159,7 @@ When running the upcoming commands, make sure to replace `$PATH_TO_YOUR_CONTAINE
 zenml integration install gcp
 
 # Create the stack and its components
-zenml container-registry register gcp_registry --uri=$PATH_TO_YOUR_CONTAINER_REGISTRY
+zenml container-registry register gcp_registry --uri=$PATH_TO_YOUR_CONTAINER_REGISTRY --type=default
```
Comment (AlexejPenner) [inline docs/book/guides/functional-api/deploy-to-production.md:162]:
```suggestion
zenml container-registry register gcp_registry  --type=default --uri=$PATH_TO_YOUR_CONTAINER_REGISTRY
```

File: docs/book/guides/functional-api/deploy-to-production.md [hunk 4]
```diff
@@ -62,7 +62,7 @@ Let's stick with the `local_metadata_store` and a `local_artifact_store` for now
 
 ```bash
 # Make sure to create the local registry on port 5000 for it to work 
-zenml container-registry register local_registry --uri localhost:5000
+zenml container-registry register local_registry --uri=localhost:5000 --type=default
```
Comment (AlexejPenner) [inline docs/book/guides/functional-api/deploy-to-production.md:65]:
```suggestion
zenml container-registry register local_registry --type=default --uri=localhost:5000 
```

**Review by AlexejPenner — COMMENTED** [review]
@htahir1  Like this?

File: examples/kubeflow/setup.sh (hunk 1 above)
Comment (htahir1) [reply examples/kubeflow/setup.sh:6]: Yes!

**Review by htahir1 — APPROVED** [review]
LGTM!

**Pull request merged**